### DISCUSSION TITLE
playwright trace link gives a 404 when followed

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#3386] [Artillery]
when I click on any of the traces in this run, then click on the open trace in new tab buttong - which before was giving me a playwright like page where I could check the selectors of elements on the page etc. I am now getting a new page with a 404 and something like this
`
This trace.playwright.dev page can’t be found
`

**Comment by hassy** [comment]
looking into it @zoe-jobson-IL, thanks for the bug report!
